src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -455,6 +460,45 @@ protected function move(Request $request, $movement)
     }
 
     /**
+     * @param Request $request
+     * @param string $transition
+     * @param string $graph
+     *
+     * @return RedirectResponse
+     */
+    public function updateStateAction(Request $request, $transition, $graph = null)
```

Comment:
Let's make the ``$graph`` required, so that we do not add this nasty protected property.